README.md

```diff
@@ -19,6 +19,7 @@
 * :speaker: **Talk rooms!** Create an associated Talk room when booking a meeting with just one click
 * :calendar: **Appointment booking** Send people a link so they can book an appointment with you [using this app](https://apps.nextcloud.com/apps/appointments)
 * :see_no_evil: **We’re not reinventing the wheel!** Based on the great [c-dav library](https://github.com/nextcloud/cdav-library), [ical.js](https://github.com/mozilla-comm/ical.js) and [fullcalendar](https://github.com/fullcalendar/fullcalendar) libraries.
+* :paperclip: **Attachments!** Add, upload and view attachments
```

Comment:
Maybe this would be better one point up? So above we are not reinwenting the wheel?